cadquery/assembly.py

```diff
@@ -488,11 +490,22 @@ def save(
         elif exportType == "VRML":
             exportVRML(self, path, tolerance, angularTolerance)
         elif exportType == "GLTF":
-            exportGLTF(self, path, True, tolerance, angularTolerance)
+            # Handle the binary option for GLTF export
+            binary = True
+            path_parts = path.split(".")
+            if len(path_parts) > 0 and path_parts[-1] == "gltf":
+                binary = False
```

Comment:
Pending discussion/conclusion of #1419, the binary handling could be moved to `exportGLTF`.

Reply:
@lorenzncode I've restructured this PR to take care of #1419 and #1420

Reply:
With the PR branch this is working:
`cq.exporters.assembly.exportGLTF(assy, "assy.glb")`

This fails:
`assy.save("assy.glb")`